### PR TITLE
[FEATURE] Add example for sending HTTP request with Guzzle

### PR DESCRIPTION
**Pull request by brotkrueml**
This example will be used as code snippet for
https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Http/Index.html#basic-usage

The main part is the MeowInformationRequester class which requests a cat information
from a public API. To hold the example small only the needed code is
available in this class. To really work with the MeowInformationRequester a
MeowInformationCommand is introduced which can be called with
`vendor/bin/typo3 examples:meow` and displays a random information piece
about cats.

Resolves: #108

### CODE REVIEW

File: Classes/Http/CatFactRequester.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        }
+
+        if ($response->getHeaderLine('Content-Type') !== 'application/json') {
+            throw new \RuntimeException(
+                'No JSON returned'
+            );
+        }
+
+        // Get the content as a string on a successful request
+        return json_decode($response->getBody()->getContents(), true)['fact1']
```
Comment (brotkrueml) [inline Classes/Http/CatFactRequester.php:49]:
```suggestion
        return json_decode($response->getBody()->getContents(), true)['fact']
```

Comment (brotkrueml) [reply Classes/Http/CatFactRequester.php:49]: Grr, why can't I apply the change? Forgot to remove the wrong array key from testing.

**Comment by brotkrueml** [comment]
Interesting, as I don't like and use abbreviations that relation haven't come to my mind ;-) I just thought about the "Requester" suffix, if there may be some other, better word for it.

I can swich to a Chuck Norris API, so `ChuckNorrisJokeRequester` may be the way:
http://api.icndb.com/jokes/random

However, I can't either adjust my PR on GUI neither locally, another push is forbidden for me, maybe some access thingy.

**Comment by DavidBruchmann** [comment]
> I can swich to a Chuck Norris API, so ChuckNorrisJokeRequester may be the way

Great idea 😂

**Comment by DavidBruchmann** [comment]
`ChuckRequester` would be enough, but it's your party :laughing:

**Comment by brotkrueml** [comment]
> `ChuckRequester` would be enough, but it's your party laughing

Stick to the cat but renamed command and requester to MeowInformation. Don't liked the chuck jokes.

**Review by linawolf — APPROVED** [review]

**Review by mabolek — CHANGES_REQUESTED** [review]

**Review by linawolf — APPROVED** [review]

**Comment by linawolf** [comment]
@brotkrueml please resolve the conflicts

**Comment by linawolf** [comment]
There is still a conflict...

**Review by linawolf — APPROVED** [review]

**Pull request merged**